src/components/heading/heading.scss

```
@@ -59,15 +59,26 @@ $max-width-permalink-shift: math.div($min-width-permalink-shift * 16 - 1, 16);
 /// The default font weight for headings is normally defined by heading level.
 /// This modifier will change the heading to font-weight.$light. We do this to
 /// maintain the importance of headings without drowning out other content.
+///
+/// 1. Restore default font weight for bold/emphasis elements nested within
+///    light headings. Without this, `b` and `strong` elements will only be one
```

Unsure if my suggestion is accurate, feel free to ignore if I'm misunderstanding. 👍🏽 
```suggestion
///    light and medium headings. Without this, `b` and `strong` elements will only be one
```